### ISSUE TITLE
F-Droid Versions Messed Up

### ISSUE THREAD

**Issue by OkyDooky** [lesspass/lesspass#745]
So, my device is prompted for an update, then I get notified the install failed, so I looked and it said the update was incompatible with my device. Well, turns out that one of the versions actually is compatible, but it is on the F-Droid Archive repository (even though it has the same upload date as the others) and, it seems, because of that, I am still prompted to "update" to the "latest version."
Here is a screenshot (excuse the AMOLED dark mode):
![Screenshot_2022-12-26_20-58-51](https://user-images.githubusercontent.com/59938024/209614616-e8b6b63d-9023-4b22-b6c1-00d4e70286d2.jpg)

**Comment by guillaumevincent** [comment]
Hello @OkyDooky,

I'm sorry for this. I will check with FDroid.

@licaon-kter any idea ?

**Comment by licaon-kter** [comment]
9.7.3 is installed... No issue there.

Also @OkyDooky pls in F-Droid Settings, disable "show incompatible" ;)

**Comment by guillaumevincent** [comment]
@licaon-kter thank you, yes closing this one

**Comment by OkyDooky** [comment]
@guillaumevincent No problem. Minor inconvenience, but more a technical oddity I thought you would want brought to your attention, especially for other users with (older) devices like mine.
@licaon-kter Yes, I manually installed it, then took the screenshot. I apologize for the imperfection. I had the "show incompatible versions" enabled for another situation a while back and didn't disable it, but it helped me catch this.
Yes, to see the history of apps (sizes, release frequency) and general access.
No, this was a valid issue; wrong repro to open it in, at worst. I am not yet familiar with repprting things in the F-Droid side of things and I figured guillaumevincent would need to sort things out with your team, anyways. So, I reported it here. The world wouldn't end, but this could interrupt the update process until the next version for me (or anyone else), if I didn't have those settings enabled on this device.
Thanks for taking care of this, guys. 🙏
